common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -651,10 +681,6 @@
         "no-regex-spaces": [
             "error"
         ],
-        "no-restricted-syntax": [
-            "error",
-            "ForInStatement"
```

Comment:
I think we should continue disallowing for-in statements

Reply:
I thought this was a duplicate of [guard-for-in](https://eslint.org/docs/rules/guard-for-in). Do you think that's insufficient?